Documentation/observability/hubble-exporter.rst

```diff
@@ -151,3 +151,86 @@ The following is a complete example of configuring Hubble Exporter.
        {"flow":{"time":"2023-08-21T12:12:18.510175415Z","verdict":"DROPPED","IP":{"source":"10.244.1.60","destination":"10.244.1.5","ipVersion":"IPv4"},"l4":{"TCP":{"source_port":44916,"destination_port":80,"flags":{"SYN":true}}},"source":{"namespace":"default","pod_name":"xwing"},"destination":{"namespace":"default","pod_name":"deathstar-7848d6c4d5-th9v2"},"node_name":"kind-kind/kind-worker","drop_reason_desc":"POLICY_DENIED"},"node_name":"kind-kind/kind-worker","time":"2023-08-21T12:12:18.510175415Z"}
+
+
+Dynamic exporter configuration
+==============================
+
+Standard hubble exporter configuration accepts only one set of filters and
+requires cilium pod restart to change config. Dynamic flow logs allow configuring
+multiple filters at the same time and saving output in separate files.
+Additionally it does not require cilium pod restarts to apply changed configuration.
+
+**Dynamic Hubble Exporter** is enabled with Config Map property. It is disabled
+until you set a file path value for ``hubble-flowlogs-config-path``.
+
+.. include:: ../installation/k8s-install-download-release.rst
+
+Install cilium with dynamic exporter enabled:
+
+.. code-block:: shell-session
+
+   helm install cilium |CHART_RELEASE| \
+      --set hubble.enabled=true \
+      --set hubble.export.dynamic.enabled=true
+
+Wait for ``cilium`` pod to become ready:
+
+.. code-block:: shell-session
+
+    kubectl -n kube-system rollout status ds/cilium
+
+Verify that flow logs are stored in target files:
+
+.. code-block:: shell-session
+
+    kubectl -n kube-system exec ds/cilium -- tail -f /var/run/cilium/hubble/events.log
+
+You can change flow log settings without a need for pod to be restarted
+(changes should be reflected within 60s because of configmap propagation delay):
+
+.. code-block:: shell-session
+
+   helm upgrade cilium |CHART_RELEASE| \
+      --set hubble.enabled=true \
+      --set hubble.export.dynamic.enabled=true \
+      --set hubble.export.dynamic.config.content[0].name=system \
+      --set hubble.export.dynamic.config.content[0].filePath=/var/run/cilium/hubble/events-system.log \
+      --set hubble.export.dynamic.config.content[0].IncludeFilters[0].source_pod[0]='kube_system/*' \
+      --set hubble.export.dynamic.config.content[0].IncludeFilters[1].destination_pod[0]='kube_system/*'
+
+
+Dynamic flow logs can be configured with ``end`` property which means that it will
+automatically stop logging after specified date time. It supports the same
+field masking and filtering as static hubble exporter.
+
+For max output file size and backup files dynamic exporter reuses the same
+settings as static one: ``hubble-export-file-max-size-mb`` & ``hubble-export-file-max-backups``
+
+Sample flow logs configs:
```

Comment:
Could you add a few words explaining that it should end up in another ConfigMap?

Reply:
I change example to reflect the sample config in helm values (for consistency with other samples).